services/pypi/pypi.tester.js

```diff
@@ -215,6 +215,14 @@ t.create('python versions (valid, no package version specified)')
     })
   )
 
+t.create('python versions ("Only" and others)')
+  .get('/pyversions/uvloop/0.12.1.json')
+  .expectJSON({ name: 'python', value: '3.5 | 3.6 | 3.7' })
+
+t.create('python versions ("Only" only)')
+  .get('/pyversions/hashpipe/0.9.1.json')
+  .expectJSON({ name: 'python', value: '3' })
```

Comment:
I've a slight preference to extract this functionality to a helper function so it can be tested through unit tests, as we've done with `getLicenses` and `getPackageFormats` in `pypi-helpers.js`.

Package metadata can change, so even if these classifiers are present on these packages today, these tests don't prevent regressions and aren't self-documenting.

Reply:
I tend to agree, but I'm merely following the existing practice established in pypi.tester.js here; bigger structural changes belong in separate commits/PR's in my opinion.

Then again, FWIW, metadata on versioned releases is not directly changeable by authors, because PyPI doesn't allow a single version to be uploaded more than once.

Reply:
I see; these are tagged versions.

I'll open a new issue for the refactor and link to this example: https://github.com/badges/shields/pull/3014#discussion_r